core/src/test/scala/unit/kafka/admin/ConfigCommandTest.scala

```diff
@@ -424,12 +483,33 @@ class ConfigCommandTest extends ZooKeeperTestHarness with Logging {
 
   @Test
   def shouldAlterTopicConfig(): Unit = {
+    doShouldAlterTopicConfig(false)
+  }
+
+  @Test
+  def shouldAlterTopicConfigFile(): Unit = {
+    doShouldAlterTopicConfig(true)
+  }
+
+  def doShouldAlterTopicConfig(file: Boolean): Unit = {
+    var filePath = ""
+    val addedConfigs = Seq("delete.retention.ms=1000000", "min.insync.replicas=2")
+    if (file) {
+      val file = File.createTempFile("testParseConfigsToBeAddedForAddConfigFile", ".properties")
```

Comment:
TestUtils.tempFile().  we can also add a new util method  `TestUtils.tempFile(String data)` and use at both the places.